code/__DEFINES/mobs.dm

```diff
@@ -538,6 +538,13 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define SHRIKE_FLAG_PAIN_HUD_ON		(1<<0)
 #define SHRIKE_CURE_HEAL_MULTIPLIER	10
 
+//Runner defines
+#define RUNNER_EVASION_STACKS				250 //How much projectile damage we auto-dodge.
+#define RUNNER_EVASION_DURATION				8 SECONDS //How long Evasion lasts.
+#define RUNNER_EVASION_DURATION_WARNING		0.7 //After what % of Evasion's time elapsed do we warn the user
+#define RUNNER_EVASION_RUN_DELAY			0.5 SECONDS //If the time since the Runner last moved is equal to or greater than this, its Evasion ends.
+#define RUNNER_EVASION_DANGER_RATIO			0.5 //If we have this % of auto-dodge damage remaining or less, warn the user.
```

Comment:
If i'm correct this means 12.5 damage remaining. That seems like less than half a single rifle bullets worth of damage. up this maybe?

Reply:
It's 125, or half of the Runner Evasion Stacks which is 250.